### ISSUE TITLE
Timeline appears RTL

### ISSUE THREAD

**Issue by Sowa0** [jerryzhou196/seepreviousratings#7]
Thank you very much for spending the time on implementing this addon.

The addon seems to inherit some RTL properties from one of my field (Arabic is written RTL). There is no reference to RTL in my CSS. The colour blocks are shown in reverse order and Card Rating History is shown on the right of the timeline.
![image](https://user-images.githubusercontent.com/9091449/128039599-71d4fb92-d856-44e9-92ab-7d5e0b464bbd.png)

As expected, I have no issue with other cards.
![image](https://user-images.githubusercontent.com/9091449/128039476-4b3bfed9-9dbb-44cc-838d-2629762e72f0.png)

Anki version: 2.1.35

**Comment by jerryzhou196** [comment]
Thank you so much for bringing this up. I will look into trying to sepearte the CSS components for the timeline from the card so that they stop leeching into the addon. (it's been a problem with lots of people too) I really appreciate that you are taking the time out of your day to report this - will keep you posted! :)

**Comment by Sowa0** [comment]
No worries, it is an amazing addon and it does the job pretty well. I am not too fussed about it, I just wanted to make sure it is reported since you have been asking for feedback.

**Comment by jerryzhou196** [comment]
Okay just implemented @saxoncameron idea of adding the `!important` tag to all the CSS styles. Tell me if it fixes your card styling issues - Cheers!

**Comment by Sowa0** [comment]
Was the update pushed yesterday? I did receive an update but long before your comment. So far, the same issue remains.

After some testing, we can safely assume that the HTML direction for all objects is inherited from the card's first field (I added a table at the bottom of my HTML and it showed RTL as well).

My solution is rather inelegant `<div id = "legend-container", style ="direction: ltr">` where the direction is set at container level.
It would be better to grab the interface language and identify if the addon should be displayed LTR or RTL. Though I doubt that most addons accommodate for this.
